sharness/t0020-service-basic-commands.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+test_description="Test service startup and init functionality"
+
+. lib/test-lib.sh
+
+test_expect_success "ipfs setup" '
+    test_ipfs_init &&
+    test_have_prereq IPFS_INIT
```

Comment:
Does this apply to t0010 as well?  I am thinking no because it does not make use of any daemons but perhaps we want consistency.

Also do you have any objections to:

``test_expect_success "ipfs cluster setup" '
        if ! test_have_prereq IPFS; then
            skip_all="..."
            test_done
        fi '
``
or is there a better way to skip all subsequent tests by including the prereqs as the 1st arg to test_expect_success as you wrote above?